auth0_flutter/lib/auth0_flutter_web.dart

```diff
@@ -19,6 +19,15 @@ class Auth0Web {
   Auth0Web(final String domain, final String clientId)
       : _account = Account(domain, clientId);
 
+  /// Get the app state that was provided during a previous call
+  /// to [loginWithRedirect].
+  ///
+  /// This method should be called after calling [onLoad].
+  ///
+  /// This getter can only be called once,
+  /// after which it will return `null` for subsequent invocations.
+  Future<Object?> get appState => Auth0FlutterWebPlatform.instance.appState;
```

Comment:
What is the purpose of having `appState` return `null` after the first invocation? I don't think the `appState` from the SPA JS behaves this way...

Reply:
I can change that, sure. We should be porting the JS API 1-1, so this should match indeed.